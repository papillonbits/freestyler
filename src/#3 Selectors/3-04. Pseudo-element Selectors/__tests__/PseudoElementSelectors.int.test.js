import { regular } from './PseudoElementSelectors.int.story'

describe('<PseudoElementSelectors />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
