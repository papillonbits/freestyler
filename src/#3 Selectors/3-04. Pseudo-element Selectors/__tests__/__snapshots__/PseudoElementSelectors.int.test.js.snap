// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PseudoElementSelectors /> Render must match regular() 1`] = `
<div>
  <div
    class="pseudo-element-selectors"
  >
    <h1>
      Pseudo-element Selectors
    </h1>
    <p>
      <code>
        first-letter
      </code>
       selects the first letter, and
      <code>
        first-line
      </code>
       selects the first line of a terminal block element, like this paragraph.
    </p>
    <div>
      <span>
        Pseudo-element selectors do not work on inline elements.
      </span>
    </div>
    <dl>
      <dt>
        Pseudo-element selectors do not work on structural block elements.
      </dt>
    </dl>
  </div>
</div>
`;
